rules/csharp/security/binary-formatter.yml

```
@@ -0,0 +1,12 @@
+id: binary-formatter
+language: csharp
+message: 'Avoid using BinaryFormatter, it is insecure and can lead to remote code execution'
+severity: warning
+note: >-
+  [CWE-502]: Deserialization of Untrusted Data
+  [OWASP A08:2017]: Insecure Deserialization
+  [OWASP A08:2021]: Software and Data Integrity Failures
+  [REFERENCES]
+      - https://docs.microsoft.com/en-us/dotnet/standard/serialization/binaryformatter-security-guide
+rule:
+  pattern: new BinaryFormatter()
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)